Examples/Tests/PythonWrappers/PICMI_inputs_2d.py

```diff
@@ -0,0 +1,245 @@
+        ax.annotate('PML', xy = (nx+nxpml//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (nx//2,-nzpml//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML', xy = (nx//2,nz+nzpml//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (-nxpml-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx+nxpml+nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,-nzpml-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nz-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nz+nzpml+nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        # Set extent and sliced data
+        extent = np.array([-nxg-nxpml, nx+nxpml+nxg, -nzg-nzpml, nz+nzpml+nzg])
+    else:
+        # Draw ghost regions
+        ax.axvline(x = 0       , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx      , linewidth = lw, linestyle = ls)
+        ax.axhline(y = 0       , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz      , linewidth = lw, linestyle = ls)
+        # Annotations
+        ax.annotate('ghost', xy = (-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('ghost', xy = (nx+nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('ghost', xy = (nx//2,-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('ghost', xy = (nx//2,nz+nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        # Set extent and sliced data
+        extent = np.array([-nxg, nx+nxg, -nzg, nz+nzg])
+    X = data[:,:].transpose()
+    # Min and max for colorbar
+    vmin, vmax = compute_minmax(X)
+    # Display data as image
+    im = ax.imshow(X = X, origin = 'lower', extent = extent, vmin = vmin, vmax = vmax, cmap = 'seismic')
+    # Add colorbar to plot
+    fig.colorbar(im, cax = cax)
+    # Set label for x- and y-axis, set title
+    ax.set_xlabel('x')
+    ax.set_ylabel('z')
+    ax.set_title(title)
+    # Set plot title
+    suptitle = 'PML in (x,z), 4 grids 64 x 64'
+    plt.suptitle(suptitle)
+    # Save figure
+    figname = 'figure_' + name + '.png'
+    fig.savefig(figname, dpi = 100)
+
+# Initialize fields data (unit pulse) and apply smoothing
+def init_data(data):
+    data[nx//2,nz//2] = 1.
```

Comment:
Thanks for updating this. Though, this line setting the central point to 1 is not needed since it will be overwritten below.

Reply:
Right, sorry about that. Fixed in https://github.com/ECP-WarpX/WarpX/pull/2576/commits/66ec869749ffde6c8ed047729d77f002c1d5a21b (along with other things).